packages/react-components/src/components/VideoGallery/LocalScreenShare.tsx

```diff
@@ -21,15 +30,18 @@ export const LocalScreenShare = React.memo(
     isAvailable?: boolean;
     onCreateLocalStreamView?: (options?: VideoStreamOptions) => Promise<void | CreateVideoStreamViewResult>;
     onDisposeLocalScreenShareStreamView?: () => Promise<void>;
+    hideLocalScreenShareStream?: boolean;
   }) => {
     const {
       localParticipant,
       renderElement,
       isAvailable,
       onCreateLocalStreamView,
-      onDisposeLocalScreenShareStreamView
+      onDisposeLocalScreenShareStreamView,
+      hideLocalScreenShareStream
```

Comment:
if we are specifying there is a default in the JSdocs we should make sure we are setting that here in the code as well instead of relying on undefined.